open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/nvd/Config.java

```diff
@@ -17,6 +17,7 @@
 package io.github.jeremylong.openvulnerability.client.nvd;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
```suggestion
```

Reply:
The schema allows additional properties: https://github.com/jeremylong/Open-Vulnerability-Project/blob/83189a18225029b1cb535f747e6b7662acfd0ecd/open-vulnerability-clients/src/main/resources/json/cve_api_json_2.0.schema#L371-L393